docs/posts/2024-09/ope_sec.md

```
@@ -98,7 +98,7 @@ D.奶油夹心饼干
 | :---: | :---: | :---: | :---: |
 | A.角峰做的松饼早餐卷广受罗德岛上干员们的喜爱，甚至到了去晚了就没有早餐吃的地步 | B.早露因为沉浸于工作中而错过了晚饭饭点，但古米和凛冬给她做了家乡的乌萨斯饺子 | C.伊芙利特等人为水月准备了仿蒲烧鳝鳞饭，但在去拿饭的时候被水月误以为是不喜欢自己做的饭而感到失落 | D.棘刺的特制蒜香壳虾塔帕斯中有一个特辣版本，不出意外地被极境吃到了 |
 
-![](./res/ope_sec/answer.webp){style="float:left;max-width:20%;margin-right:1em"}
+![](./res/ope_sec/answer.webp){.image-left-float style="max-width: 20%;"}
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)
null

(MD045, no-alt-text)